config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
What are the files in the certmanager directory for?

Reply:
Cert-manager is used for web hooks, currently its disabled if we want to use webhooks in the future, then we can enable it. It's generated by default via Kube-builder.